src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -536,6 +535,38 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </>
   );
 
+  const renderParentIdInput = () => (
```

Comment:
I don't see a point of it being inside a function. You can just inline it.

Reply:
I just did it the same way it's done in staging. This PR is just a revert of that part being removed. i guess it was done this way for better visibility. We previously had everything inline here. But the file is super huge so I guess it was the way to make it easier to read.